activerecord/test/schema/postgresql_specific_schema.rb

```diff
@@ -46,14 +46,12 @@
 "
   end
 
-  if supports_identity_columns?
```

Comment:
@fatkodima 

I might be doing something wrong, but on `main` this test does not pass, as the `supports_identity_column?` is not defined in this schema file context.

When I removed the method, everything works. The schema objects are created (the table with the identity column, etc.), and then the test passes. Used this:

```rb
ARCONN=postgresql ruby -I activerecord/test/ \
activerecord/test/cases/adapters/postgresql/create_unlogged_tables_test.rb
```

I'm wondering if I'm doing something wrong, or this method was intended to be defined at a different level than for example the method within create_table that checks whether virtual columns are supported.

I noticed when the schema load happens, it loads:

* A non-adapter-specific schema
* Then a adapter specific schema (PostgreSQL here)

Wondered whether the conditional method was necessary.

I think it was added for Version 10 here recently: https://github.com/rails/rails/pull/49598